db/factories/AngelTypeFactory.php

```diff
@@ -27,6 +27,7 @@ public function definition(): array
             'no_self_signup'          => $this->faker->boolean(),
             'show_on_dashboard'       => $this->faker->boolean(),
             'hide_register'           => $this->faker->boolean(),
+            'hide_on_shift_view'      => $this->faker->boolean(),
```

Comment:
I'm thinking of how to make this shorter and / or more precise (less "technical"), eventually `hide_shift_overview`? Or `hide_shift_list`? The `on` seems redundant (or we should rename the `hide_register` later too to `hide_on_registration` (or `signup` when we are on it). ^^

Reply:
hide_on_shift_view!!!!!